.travis.yml

```diff
@@ -7,5 +7,8 @@ rvm:
   - 2.3.7
```

Comment:
Can probably get rid of some of the older 2.3.x versions and just check this more recent 2.3.7?